client/src/components/SystemMonitor/SystemMonitorCells.jsx

```diff
@@ -1,15 +1,23 @@
 import React from 'react';
 import { Pill } from '_common';
 import { shape, string, bool, number } from 'prop-types';
+import { Link } from 'react-router-dom';
 
 const CELL_PROPTYPES = {
   cell: shape({
     value: string.isRequired,
   }).isRequired,
 };
 
-export const Display = ({ cell: { value } }) => (
-  <strong className="wb-text-primary">{value}</strong>
+export const Display = ({ cell: { row } }) => (
+  <strong>
+    <Link
+      className="wb-text-primary"
```

Comment:
This is a link now, not text. It should be colored as such. This change will do that.

```suggestion
      className="wb-link"
```

Reply:
That this need be managed by a class per link is inefficient to begin with.  I am sorry I have yet to remove these kind of mud puddles that has been around since I joined.

Reply:
Committed this tiny fix, so I can approve on behalf of Design.